docs/_docs/04_community/contributing.md

```diff
@@ -150,16 +149,50 @@ to include the following:
    test output so we can verify that everything is working.
 1. Any notes on backwards incompatibility or downtime.
 
+#### Validate the Pull Request for Azure Platform
+
+After [creating a pull request](https://help.github.com/articles/creating-a-pull-request/), please follow the below guideline for [Azure Platform](https://azure.com)
+
+##### To run the CI Pipeline on forked repo

```

Comment:
```suggestion
##### Run the Azure CI Pipeline on your forked repo

We have a separate CI pipeline for Azure code. To run it on a forked repo:

```

Reply:
Do we expect all Azure contributors to run the tests this way? Or are these instructions specifically for MS employees?

Reply:
we recommend them to follow this guideline if they have an active Azure subscription.

it'll shorten the develop-test-fail-fix loop.

'cause they can run the pipeline on their repo, fix the issues, create a pr.

it'll be much faster than, create a pr, check the result, fix the issues, update the pr.